spk/sudo/Makefile

```diff
@@ -0,0 +1,25 @@
+SPK_NAME = sudo
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/sudo.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = frv
+DESCRIPTION = "Sudo allows a system administrator to delegate authority to give certain users (or groups of users) the ability to run some (or all) commands as root or another user while providing an audit trail of the commands and their arguments."
+RELOAD_UI = no
+DISPLAY_NAME = sudo
+STARTABLE = no
+BETA = 1
+
+HOMEPAGE   = http://www.sudo.ws
+LICENSE    = ISC-style license (TODO: check the compliance of the licence)
```

Comment:
Where does this come from? If nothing else, put a link to the official license in here: (seems to be http://www.sudo.ws/sudo/license.html), because that's what we're referring to here.